### PR TITLE
Global threat updates to CC 3.16 and next versions

### PR DESCRIPTION
**Pull request by girishshankaran**
The global threat updates only made it to 3.15 version. This PR is to push the changes to CC 3.16 and next versions.

Product Version(s):

Issue:

Link to docs preview:

SME review:
- [ ] An SME has approved this change. 

DOCS review:
- [ ] A member of the docs team has approved this change.

Additional information:

### CODE REVIEW

**Comment by girishshankaran** [comment]
@bmckercher123 Can you please review this?

File: calico-cloud_versioned_docs/version-3.16/threat/suspicious-ips.mdx [hunk 1]
```diff
@@ -37,7 +35,7 @@ Privileges to manage GlobalThreatFeed and GlobalNetworkPolicy.
 
 ### Recommended
 
-We recommend that you turn down the aggregation of flow logs sent to Elasticsearch for configuring threat feeds. If you do not adjust flow logs, {{prodname}} aggregates over the external IPs for allowed traffic, and threat feed searches will not provide useful results (unless the traffic is denied by policy). Go to: [FelixConfiguration](../reference/resources/felixconfig.mdx) and set the field, **flowLogsFileAggregationKindForAllowed** to **1**.
+We recommend that you turn down the aggregation of flow logs sent to Elasticsearch for configuring threat feeds. If you do not adjust flow logs, Calico Enterprise aggregates over the external IPs for allowed traffic, and threat feed searches will not provide useful results (unless the traffic is denied by policy). Go to: [FelixConfiguration](../reference/resources/felixconfig.mdx) and set the field, **flowLogsFileAggregationKindForAllowed** to **1**.
```
Comment (ctauchen) [inline calico-cloud_versioned_docs/version-3.16/threat/suspicious-ips.mdx]:
```suggestion
We recommend that you turn down the aggregation of flow logs sent to Elasticsearch for configuring threat feeds. If you do not adjust flow logs, {{prodname}} aggregates over the external IPs for allowed traffic, and threat feed searches will not provide useful results (unless the traffic is denied by policy). Go to: [FelixConfiguration](../reference/resources/felixconfig.mdx) and set the field, **flowLogsFileAggregationKindForAllowed** to **1**.
```

Comment (ctauchen) [reply calico-cloud_versioned_docs/version-3.16/threat/suspicious-ips.mdx]: Same fix in other file, if applicable.

**Review by ctauchen — APPROVED** [review]
LGTM, with one fix needed.

File: calico-cloud_versioned_docs/version-3.16/threat/suspicious-ips.mdx (hunk 1 above)
Comment (girishshankaran) [reply calico-cloud_versioned_docs/version-3.16/threat/suspicious-ips.mdx]: Good catch, Chris! It's fixed

**Comment by ctauchen** [comment]
LGTM

**Pull request merged**